Xamarin.Forms.Core/Color.cs

```diff
@@ -31,8 +31,19 @@ public bool IsDefault
 		}
 
 		[EditorBrowsable(EditorBrowsableState.Never)]
-		public static void SetAccent(Color value) => Accent = value;
-		public static Color Accent { get; internal set; }
+		public static void SetAccent(Color value) => s_accent = value;
+
+		static Color s_accent;
+		public static Color Accent
+		{
+			get
+			{
+				if (Device.RuntimePlatform == Device.Android && s_accent.IsDefault)
```

Comment:
This is getting material accent color if Core accent color is default. Since `SetAccent` is a public method, it is possible that one sets accent color to default in which case we'd re-set it to theme-specific accent color. However, I'm not sure why anybody would do this on purpose.